newrelic-agent/src/main/java/com/newrelic/api/agent/NewRelicApiImplementation.java

```diff
@@ -56,7 +62,7 @@ public NewRelicApiImplementation() {
      */
     @Override
     public void noticeError(Throwable throwable, Map<String, ?> params) {
-        noticeError(throwable, params, false);
+        noticeError(throwable, params, isExpectedErrorConfigured(throwable.getClass().getName(), throwable.getMessage()));
```

Comment:
I will need to add this check here as well:
`noticeError(throwable, params, throwable != null && isExpectedErrorConfigured(throwable.getClass().getName(), throwable.getMessage()));`